### PR TITLE
Landing page pom/feature/sd full implementation of working tests

### PR DESCRIPTION
**Pull request by Bryar-Frank**

### CODE REVIEW

File: cucumber-selenium-tests/src/test/java/com/skillstorm/PageObjects/LandingPage.java [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+import com.skillstorm.PageObjects.Components.Navbar.LandingNavbar;
+import com.skillstorm.PageObjects.Interfaces.Component;
+
+public class LandingPage extends Page {
+//#region Static fields
+
+    //Naming System
+    public static final String CMP_LANDING_NAVBAR_NAME = "Landing Navbar";    
+    public static final String CMP_FOOTER_NAME = "Footer";
+    public static final String NAME_FEATURES_CONTAINER = "Features";
```
Comment (achuggi1) [inline cucumber-selenium-tests/src/test/java/com/skillstorm/PageObjects/LandingPage.java:22]: Low impact: difference in naming conventions in the same file.

File: cucumber-selenium-tests/src/test/java/com/skillstorm/StepDefinitions/SDLanding.java [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+
+public class SDLanding {
+
+    private WebDriver driver;
+    private WebDriverWait wait;
+    private Navigator navigator;
+    private LandingPage page;
+    private User user = new User(UserType.PERSISTANT, "joseph.sam@gmail.com", "password1");
```
Comment (achuggi1) [inline cucumber-selenium-tests/src/test/java/com/skillstorm/StepDefinitions/SDLanding.java]: Medium impact: Potential modifications to given user that other devs may be working with. Suggestion: use a new local user.

**Review by achuggi1 — CHANGES_REQUESTED** [review]
User "joseph.sam@gmail.com" in SDLanding.java has potential use in various branches/repos, and modification to this profile could cause problems in the long-term. Please update to use a different profile.

File: cucumber-selenium-tests/src/test/java/com/skillstorm/PageObjects/LandingPage.java (hunk 1 above)
Comment (Bryar-Frank) [reply cucumber-selenium-tests/src/test/java/com/skillstorm/PageObjects/LandingPage.java:22]: fixed

File: cucumber-selenium-tests/src/test/java/com/skillstorm/StepDefinitions/SDLanding.java (hunk 2 above)
Comment (Bryar-Frank) [reply cucumber-selenium-tests/src/test/java/com/skillstorm/StepDefinitions/SDLanding.java]: fixed to use persistent user for now that I can set locally

**Review by achuggi1 — APPROVED** [review]
LGTM

**Review by msabas1 — APPROVED** [review]

**Review by jmensah365 — APPROVED** [review]

**Pull request merged**